### PR TITLE
Axente octavian branch

### PR DESCRIPTION
**Pull request by Octimus7**

### CODE REVIEW

File: AxenteOctavian/AxenteOctavian.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This file was created by Axente Octavian
```
Comment (CrisFagetan) [inline AxenteOctavian/AxenteOctavian.txt:1]: Hi there!

File: AxenteOctavian/MPI_HelloWorld.c [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#include <mpi.h>
+#include <stdio.h>
+
+int main(int argc, char** argv) {
+    // Initialize the MPI environment
+    MPI_Init(NULL, NULL);
+
+    // Get the number of processes
+    int world_size;
```
Comment (CrisFagetan) [inline AxenteOctavian/MPI_HelloWorld.c:9]: this should be worldSize - just because :))

**Comment by Octimus7** [comment]
Perfect

**Review by CrisFagetan — APPROVED** [review]
Very good, very good!

**Pull request merged**